content/blog/asyncapi_codegen_scst.md

```diff
@@ -0,0 +1,186 @@
+---
+type: blog
+featured: true
+cover: /images/posts/asyncapi-codegen_pic-00.webp
+weight: 100
+title: "AsyncAPI Code Generation: Microservices Using Spring Cloud Stream"
+date: 2020-04-28 17:39:34 UTC
```

Comment:
```suggestion
date: 2020-05-28T10:00:00+01:00
```

it is set to past as we still want it to appear in the preview, but anyway has to be fixed as date format was wrong